pkgs/top-level/python-packages.nix

```diff
@@ -4903,7 +4903,7 @@ self: super: with self; {
 
   formulaic = callPackage ../development/python-modules/formulaic { };
 
-  foundationdb71 = callPackage ../servers/foundationdb/python.nix { foundationdb = pkgs.foundationdb71; };
```

Comment:
This seems to be missing a throw alias?

Reply:
https://github.com/NixOS/nixpkgs/pull/369145/files#diff-4878bf3fa499d7d0bab51ebe52427ba7ee884458535a7bbae5e31f245994dbf8

Reply:
Huh, must've missed that. Feel free to mark this as resolved.